### PR TITLE
NO-JIRA Add proxy forwarding doc

### PR DESCRIPTION
**Pull request by brusdev**

### CODE REVIEW

File: docs/user-manual/en/web-server.md [hunk 1]
```diff
@@ -89,3 +89,19 @@ Here is an example configuration:
    <request-log filename="${artemis.instance}/log/http-access-yyyy_MM_dd.log" append="true" extended="true"/>
 </web>
 ```
+
+## Proxy Forwarding
+
+The proxy and load balancers usually supports `X-Forwarded` headers
```
Comment (stephan2012) [inline docs/user-manual/en/web-server.md:95]: … usually _support_ …

**Review by stephan2012 — COMMENTED** [review]
LGTM. Thanks!

File: docs/user-manual/en/web-server.md (hunk 1 above)
Comment (brusdev) [reply docs/user-manual/en/web-server.md]: thanks, I have just pushed a commit to fix the typo

**Comment by clebertsuconic** [comment]
@brusdev I'm confused.. the PR title says NO_JIRA Adding doc, and the commit is fixing something... was that a mistake?

**Comment by brusdev** [comment]
@clebertsuconic the commit summary was wrong, I have just fixed it.

**Pull request closed without merging**